styles/scrollview/_layout.scss

```diff
@@ -0,0 +1,132 @@
+@import '../icons/main';
+@import '../button/layout';
+
+@include exports('scrollview/layout') {
+    $scrollview-page-button-size: 8px;
+    $scrollview-arrow-icon-size: 4em;
+
+    .k-scrollview-wrap {
+        position: relative;
+        overflow: hidden;
+    }
+
+    .k-scrollview {
+        list-style-type: none;
+        position: absolute;
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        height: 100%;
+        cursor: default;
+
+        img,
+        li {
+            -webkit-user-select: none;
+            -khtml-user-select: none;
+            -moz-user-select: none;
+            -o-user-select: none;
+            user-select: none;
```

Comment:
The distribution files are post-processed with autoprefixer, so you can remove the prefixed properties.